### PR TITLE
Fix install

### PR DESCRIPTION
**Pull request by AnaelMobilia**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | none

  - Fix install which silently break
  - Fix bad CSS & JS URL
Need to be backported at least to 9.3 ;-)

### CODE REVIEW

**Comment by trasher** [comment]
>  Need to be backported at least to 9.3 ;-)

Fixes must be done on the lower bugfixes branch targeted, and will be reported on higher branches. If any fix needs a specific version for any branch, several distinct PR must be opened.

**Comment by AnaelMobilia** [comment]
@trasher I can't change the target of the PR. Have you rights for that ?

Regards,
Anael

**Comment by trasher** [comment]
Sure. 9.2/bf or 9.3/bf?

File: inc/toolbox.class.php [hunk 1]
```diff
@@ -1045,14 +1045,6 @@ static function commonCheckForUseGLPI() {
          $error = $suberr;
       }
 
-      //database version check --is it posible?
```
[the remark this answers is not in the document]
Comment (trasher) [reply inc/toolbox.class.php]: This method is not only used for install; it may be conditionned, but not removed

**Comment by AnaelMobilia** [comment]
@trasher 9.3 please :)

File: inc/toolbox.class.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnaelMobilia) [reply inc/toolbox.class.php]: The change was made. Due to a windows filecase issue, I can't rebase to 9.3 this PR... I integrated it on #3988.

Regards

**Comment by AnaelMobilia** [comment]
Due to a windows filecase issue, I can't rebase to 9.3 this PR... I integrated it on #3988.

Regards

**Pull request closed without merging**